src/bot/root_pythia_cogs.py

```diff
@@ -63,7 +63,30 @@ async def resume(self, ctx):
             "Resumed successfully from idle state, requests can be sent again."
         )
 
-    # TODO: add a add_users command that would accept a list of ids
+    @commands.before_invoke(log_command_call)
+    @commands.command(name="addusers")
+    async def addusers(self, ctx, *args):
+        self.logger.info(f"command `addusers` received: {len(args)} arguments:")
+        for i in range(len(args)):
+            try:
+                user_id = int(args[i])
+                user = await self.dbmanager.add_user(user_id)
+            except ValueError as value_err: # Error coming from the int() cast
+                self.logger.error(f"command `addusers` received: {value_err}")
+                await ctx.message.channel.send(f"invalid argument received: an user_id is expected, ignoring it for now...")
```

Comment:
```suggestion
                await ctx.message.channel.send(f"invalid argument received: `{i}`; a UserID is expected, ignoring it...")
```
or something like that?
- I suggest `UserID` for consistency with other logs (see below in `if user is None`)
- 'a~~n~~ user', I had my doubt so I [checked](https://english.stackexchange.com/questions/105116/is-it-a-user-or-an-user)

Reply:
You are right, it has been changed with commit https://github.com/iScsc/RootPythia/pull/27/commits/115ad9d4bf9043fdbc49409c18fbcdce50c89a56

Reply:
perfect!